### ISSUE TITLE
Bean Validation constraints not reflected in generated model

### ISSUE THREAD

**Issue by simasch** [vaadin/hilla#852]
It looks like there is a regression in Hilla 2. Bean Validation generation was working in Hilla 1.

I have an example like in https://hilla.dev/docs/data-binding/binder-validation#specifying-constraints

```java
@Email
@Nonnull
private String email;
```

But this is not generated in the model!

```ts
get email(): StringModel_1 {
      return this[_getPropertyModel_1]("email", StringModel_1, [false]) as StringModel_1;
}
```

I've also tried Length

```java
@Nonnull
@Length(min = 3, max = 50)
private String firstName;
```

IMO this is a huge blocker and should be fixed ASAP

**Comment by simasch** [comment]
I've also tried @Length

```java
@Nonnull
@Length(min = 3, max = 50)
private String firstName;
```

Looks like the generation is broken!

**Comment by simasch** [comment]
I just checked with 1.3.7 and there everything works as expected:

  ```
get email(): StringModel {
    return this[_getPropertyModel]('email', StringModel, [true, new NotBlank(), new Email()]);
  }
```

**Comment by simasch** [comment]
Demo project https://github.com/simasch/javaland-meets-hilla

**Comment by platosha** [comment]
Thanks for reporting. This is a duplicate of #830, so let me close this issue. Meanwhile, I'm working on a fix for #830.
